apiary.apib

```diff
@@ -516,6 +516,23 @@ Get details for a single contact.
             + remarks: `First contact at expo` (string)
             + added_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
+            + custom_fields (array)
+                + (object)
```

Comment:
Would it be possible to move this to a shared datastructure?

Reply:
Sure. Normally I'm a against moving too much to a data structure. But this feels like a prime candidate :)

Reply:
I'll follow this one up